File: Javascript/6_Linear Search/Linear Search.js
function search(array, x) {
    for(var i = 0; i < array.length; i++){
        if(array[i] == x){
            return i;
        }
    }
    return -1;
}

var array = [6, 7, 3, 42, 9];
console.log(search(array, 42));

File: C#/6_Linear Search/Linear Search.cs
using System;

namespace Algorithm {

    class LinearSearch {

        public static int search(int[] arr, int x) {
            for(int k = 0; k < arr.Length; k++) {
                if(arr[k] == x) {
                    return k;
                }
            }
            return -1;
        }

        public static void Main(string[] args) {
            int[] arr = { 4, 2, 6, 42 };
            Console.Write(search(arr, 42));
        }
    }
}